src/breadcrumb/src/BreadcrumbItem.tsx

```diff
@@ -25,20 +34,31 @@ export default defineComponent({
       return () => null
     }
     const { separatorRef, mergedClsPrefixRef } = NBreadcrumb
+    const htmlTag = computed(() => (href ? 'a' : 'span'))
+    const ariaCurrent = computed(() => (href && isCurrent ? 'location' : null))
```

Comment:
I think we should check window location instead and don't need user to pass this prop.
In most cases user would just do the same thing.

Reply:
Yes, I think 90% of the case the devs are using a lib for the routing (vue-router) and use their own link component that handles this automatically (`aria-current`). But yes we can remove this prop to handle that automatically if the user passes an `href`

Reply:
Yep, I think it's meaningless to set `isCurrent = true` if anchor's href doesn't match the current location and vice versa.

Reply:
One possible problem of this approach (`window.location`) is we have now the responsibility to handle location changes and set the right aria attrs depending on that. This kind of increases the complexity of this component. 
I made a possible implementation of this. But I'm wondering if it's worth handling that compare to having a prop (`isCurrent`). Especially because these props `href` and `isCurrent` won't be used that much as the user will use `router-link` or `nuxt-link` most of the time.